<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>30</y>
      <width>252</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonConnect">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="State">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>系统状态:未连接</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>智慧大棚测控系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCustomPlot" name="charts_1" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>531</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="charts_2" native="true">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>170</y>
      <width>481</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mode_bt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>切换模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="mode_lb">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>手动</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="device1_bt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备1</string>
    </property>
   </widget>
   <widget class="QLabel" name="device1_lb">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="device2_bt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备2</string>
    </property>
   </widget>
   <widget class="QLabel" name="device2_lb">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="device3_bt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="device4_bt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备4</string>
    </property>
   </widget>
   <widget class="QLabel" name="device4_lb">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="device3_lb">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>节点1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>110</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>节点2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="editLog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>411</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="record_bt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>770</y>
      <width>171</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查看历史数据</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
